.github/pull_request_template.md

```diff
@@ -1,58 +1,38 @@
 
-<!--
-Thank you for opening a pull request!  Here are some tips on creating
-a well formatted contribution.
-
-Please give your pull request a title like "[component]: [short description]"
-
-This is the format for commit messages:
 
-"""
-[component]: [short description]
+<!--
+  - Please give your pull request a title like
 
-[A longer multiline description]
+      [component]: [short description]
 
-Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
-Signed-off-by: [Your Name] <[your email]>
-"""
+  - Please use this format for each git commit message:
 
-The Signed-off-by line is important, and it is your certification that
-your contributions satisfy the Developers Certificate or Origin.  For
-more detail, see SubmittingPatches.rst.
+      [component]: [short description]
 
-The component is the short name of a major daemon or subsystem,
-something like "mon", "osd", "mds", "rbd, "rgw", etc. For ceph-mgr modules,
-give the component as "mgr/<module name>" rather than a path into pybind.
+      [A longer multiline description]
 
-For more examples, simply use "git log" and look at some historical commits.
+      Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
+      Signed-off-by: [Your Name] <[your email]>
 
-This was just a quick overview.  More information for contributors is available here:
-https://raw.githubusercontent.com/ceph/ceph/master/SubmittingPatches.rst
+    For examples, use "git log".
 
+  - The Signed-off-by line in every git commit is important; see SubmittingPatches.rst.
 -->
+
 ## Checklist
-- [ ] References tracker ticket
-- [ ] Updates documentation if necessary
-- [ ] Includes tests for new functionality or reproducer for bug
-
----
-
-<details>
-<summary>Show available Jenkins commands</summary>
-
-- `jenkins retest this please`
-- `jenkins test classic perf`
-- `jenkins test crimson perf`
-- `jenkins test signed`
-- `jenkins test make check`
-- `jenkins test make check arm64`
-- `jenkins test submodules`
-- `jenkins test dashboard`
-- `jenkins test dashboard cephadm`
-- `jenkins test api`
-- `jenkins test docs`
-- `jenkins render docs`
-- `jenkins test ceph-volume all`
-- `jenkins test ceph-volume tox`
-
-</details>
+- Tracker (select at least one)
+  - [ ] Very recent bug; references commit where it was introduced
+  - [ ] New feature (no ticket)
+  - [ ] References tracker ticket
+  - [ ] Doc update (no ticket)
+- Documentation (select at least one)
+  - [ ] Updates relevant documentation
+  - [ ] No doc update is appropriate
+- Tests (select at least one)
+  - [ ] Includes unit test(s)
```

Comment:
I'd mention something about, at least, not decreasing the current code coverage. Dashboard team is planning to introduce [Codecov](https://about.codecov.io/product/features/) during this sprint to ensure coverage doesn't shrink.

Reply:
Wouldn't that be an automatic check?  What is the point of adding it to the template?

Reply:
You're right! From what I've seen in other projects, they take the coverage report as mainly informative. I guess it's because it's not always possible/easy to keep/increase the coverage.